peakina/helpers.py

```diff
@@ -42,7 +43,7 @@ class TypeInfos(NamedTuple):
 
 
 # For files without MIME types, we make fake MIME types based on detected extension
-CUSTOM_MIMETYPES = {".parquet": "peakina/parquet"}
+CUSTOM_MIMETYPES = {".parquet": "peakina/parquet", ".geojson": "application/geo+json"}
```

Comment:
are we okay with enforcing this extension for basemaps ? cf. my comment [here](https://toucantoco.atlassian.net/wiki/spaces/TTA/pages/3498344456/PAT+248+-+Handle+geographic+data+%28basemaps%29+as+usual+datasources?focusedCommentId=3503226890#comment-3503226890)

Reply:
Is `application/geo+json` really a valid mimetype ? If not, I'd rather have `peakina/geo` to make it explicit that this is a custom mimetype. And I agree with fred's comment, the extension for geo files should probably be free (if that's possible in peakina)

Reply:
> if that's possible in peakina 

MimeType guessing is not working it seems :/